src/platform/aws/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "aws"
+version = "0.1.0"
```

@zojw you can merge/rebase main to get #112 and update this version property to "0.2.0".

```suggestion
version = "0.2.0"
```

---

done~